packs/data/pfs-season-4-bestiary.db/darkened-shadow-cannon.json

```diff
@@ -0,0 +1,145 @@
+{
+    "_id": "UT4hacQMILQiIqSn",
+    "img": "systems/pf2e/icons/default-icons/hazard.svg",
+    "items": [
+        {
+            "_id": "AMtiulRhqRUiziHz",
+            "img": "systems/pf2e/icons/actions/Reaction.webp",
+            "name": "Load the Cannons",
+            "sort": 0,
+            "system": {
+                "actionCategory": {
+                    "value": "offensive"
+                },
+                "actionType": {
+                    "value": "reaction"
+                },
+                "actions": {
+                    "value": null
+                },
+                "description": {
+                    "value": "<p><strong>Trigger</strong> The ship flies within 30 feet of Star</p>\n<p><strong>Effect</strong> The cannon rolls initiative.</p>"
+                },
+                "requirements": {
+                    "value": ""
+                },
+                "rules": [],
+                "slug": null,
+                "source": {
+                    "value": ""
+                },
+                "traits": {
+                    "custom": "",
+                    "rarity": "common",
+                    "value": []
+                },
+                "trigger": {
+                    "value": ""
+                }
+            },
+            "type": "action"
+        },
+        {
+            "_id": "ICRVUZzxSiIPifMN",
+            "img": "systems/pf2e/icons/default-icons/melee.svg",
+            "name": "Cannon Blast",
+            "sort": 0,
+            "system": {
+                "attack": {
+                    "value": ""
+                },
+                "attackEffects": {
+                    "value": []
+                },
+                "bonus": {
+                    "value": 13
+                },
+                "damageRolls": {
+                    "eZtApt9vf4mq7wLh": {
+                        "damage": "2d8+5",
+                        "damageType": "bludgeoning"
+                    }
+                },
+                "description": {
+                    "value": "<p>to creatures in a @Template[type:burst|distance:5] (@Check[type:reflex|dc:21|basic:true])</p>"
```

Comment:
Because it is listed as a strike. You target a creature, if it hits it does this damage in a burst. Don't ask me why. It was the closes thing that I could get it to be.